### PR TITLE
Update dumpling-overview.md

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | Dumpling 默认会将 Binary 数据类型以 UTF-8 格式在 CSV 文件中表示。若要以其他格式（如 HEX）导出 Binary 数据，请将参数设置为 'redshift'。这样，binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀'0x'，如将'0x61' 表示成 '61'| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | Dumpling 默认将 CSV 文件中的 Binary 数据类型以 UTF-8 格式表示。若要以其他格式（如 HEX）导出 Binary 数据，请将此参数设置为 'redshift'。这样，Binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀 '0x'，如将 '0x61' 表示成 '61'。 | '"' |
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | Dumpling 默认将 CSV 文件中的 Binary 数据类型以 UTF-8 格式表示。若要以其他格式（如 HEX）导出 Binary 数据，请将此参数设置为 'redshift'。这样，Binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀 '0x'，如将 '0x61' 表示成 '61'。 | '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | Dumpling 默认将 CSV 文件中的 Binary 数据类型以 UTF-8 格式表示。若要以其他格式（如 HEX）导出 Binary 数据，请将此参数设置为 'redshift'。这样，Binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀 '0x'，如 '0x61' 将被表示成 '61'。 | '"' |
```

File: dumpling-overview.md [hunk 3]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | Dumpling 默认将 CSV 文件中的 Binary 数据类型以 UTF-8 格式表示。若要以其他格式（如 HEX）导出 Binary 数据，请将此参数设置为 'redshift'。这样，Binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀 '0x'，如 '0x61' 将被表示成 '61'。 | '"' |
```
Comment (qiancai) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | Dumpling 默认将 CSV 文件中的 Binary 数据类型以 UTF-8 格式表示。若要以其他格式（如 HEX）导出 Binary 数据，请将此参数设置为 `'redshift'`。这样，Binary 数据将以十六进制数的形式表示，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。 | '"' |
```

Comment (qiancai) [reply dumpling-overview.md:379]: 是不是只支持  UTF-8 格式  和  HEX 格式

File: dumpling-overview.md [hunk 4]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 取值可为'snowflake', 'redshift', 'bigquery'，表示可以将源数据导出成上述数据库所需的格式存储到 CSV ，如会把 binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV ，取值可为 'snowflake'、'redshift'、'bigquery'。如设置为 'redshift'，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。| '"' |
```

Comment (lance6716) [reply dumpling-overview.md:379]: 我看 redishift 和 snowflake 是 hex 编码。bigquery 是 base64 编码

Comment (Lloyd-Pottiger) [reply dumpling-overview.md:379]: +1

Comment (hfxsd) [reply dumpling-overview.md:379]: 改好了

**Review by lance6716 — APPROVED** [review]

File: dumpling-overview.md [hunk 5]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如设置为或 `"snowflake" 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。如设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如设置为 `"snowflake" 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。如设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```

File: dumpling-overview.md [hunk 6]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake" 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```

File: dumpling-overview.md [hunk 7]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，如 `0x61`将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```

**Review by qiancai — APPROVED** [review]

File: dumpling-overview.md [hunk 8]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。| '"' |
```
Comment (qiancai) [inline dumpling-overview.md:379]: 要不要写下默认值为空代表的格式是什么

Comment (Frank945946) [reply dumpling-overview.md:379]: 默认值为空，会按  UTF-8 进行编码并导出数据，在一些情况下，binary 字符串会出现乱码的情况。

Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。默认值为空，会按 UTF-8 进行编码并导出数据，在某些情况下，Binary 字符串会出现乱码。| '"' |
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: dumpling-overview.md [hunk 9]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV，取值可为 `"snowflake"`、`"redshift"`、`"bigquery"`。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。默认值为空，会按 UTF-8 进行编码并导出数据，在某些情况下，Binary 字符串会出现乱码。| '"' |
```
Comment (qiancai) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV。该选项取值可为 '"'，`"snowflake"`、`"redshift"`、或`"bigquery"`。默认值为 '"'，表示会按 UTF-8 进行编码并导出数据。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。在某些情况下，Binary 字符串会出现乱码。| '"' |
```

File: dumpling-overview.md [hunk 10]
```diff
@@ -376,6 +376,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-separator | CSV 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | CSV 文件空值的表示 | "\\N" |
 | --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\\r\\n" 和 "\\n"，默认值为 "\\r\\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\\r\\n" |
+| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV。该选项取值可为 '"'，`"snowflake"`、`"redshift"`、`"bigquery"`。默认值为 '"'，表示会按 UTF-8 进行编码并导出数据。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。在某些情况下，Binary 字符串会出现乱码。| '"' |
```
Comment (hfxsd) [inline dumpling-overview.md:379]:
```suggestion
| --csv-output-dialect | 表示可以将源数据导出成数据库所需的格式存储到 CSV。该选项取值可为 `""`，`"snowflake"`、`"redshift"`、`"bigquery"`。默认值为 '"'，表示会按 UTF-8 进行编码并导出数据。如果设置为 `"snowflake"` 或 `"redshift"`，会把 Binary 数据类型转换成十六进制，但会丢失十六进制数的前缀 `0x`，例如 `0x61` 将被表示成 `61`。如果设置为 `"bigquery"`，会使用 base64 对 Binary 数据类型进行编码。在某些情况下，Binary 字符串会出现乱码。| `""` |
```

**Pull request merged**